jenkinsfiles/rebuild-image

```diff
@@ -40,11 +40,12 @@ pipeline {
                         env.VERSIONS_TO_REBUILD.tokenize(" ").each { version ->
                             def majorVersion = version.split('\\.')[0].toInteger()
                             if (majorVersion >= 42) {
-                                env.BASE_IMAGE = "tomcat:10.0-jre17"
+                                // THIS MUST be kept in sync with the jib.from.image in ../dhis-2/dhis-web-server/pom.xml
+                                env.BASE_IMAGE = "tomcat:10.1.30-jre17"
                             } else if (majorVersion == 41) {
```

Comment:
remove `else if (majorVersion == 41)` as it sets the same version as the `else` branch.

does this pipeline even need to know about other versions? I mean if this one rebuilds images on this branch thats all it needs to know right? if `rebuild-image` on master is the one thats used this file should not exist 🤔

Reply:
The difference is that 41 is using `*-jre17` and lower versions are using `*-jre11`.

I backported the `rebuild-image` to all supported branches initially, which might have been a bit of an oversight. However, I think that was before releasing 41 (hence `master` was 41). Once we release 42 (current `master`), the file will be on the 42 dev branch and on `master`, so I suppose we either have to keep deleting it with releases or keep backporting it to lower versions branches as is and making sure it doesn't drift.